src/routes/(infoPages)/about/news/NewsEntrySnippet.svelte

```diff
@@ -20,26 +23,31 @@
 </script>
 
 {#if title && slug}
-  <div class="ldaf-news-entry">
+  <div class={`ldaf-news-entry ldaf-news-entry--${variation}`}>
     <Link href={`/about/news/article/${slug}`}>
       <svelte:element this={headingTagByLevel[headingLevel]} class="news-title">
         {title}
       </svelte:element>
     </Link>
-    {#if subhead}
-      <p>{subhead}</p>
+    {#if variation === "homepage-listed"}
+      <div class="margin-bottom-1"><Tag>{type}</Tag></div>
+    {/if}
```

Comment:
Not crazy about how I have this set up, but I wasn't sure if reordering things using CSS would be any better.  We want the tags at the top of the item when they're listed on a truncated version on the hompage, but at the bottom otherwise.